bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerEntry.java

```diff
@@ -53,22 +54,49 @@ public long getLength() {
         return length;
     }
 
+    /**
+     * Returns the content of the entry.
+     * This method can be called only once.
+     * While using v2 wire protocol this method will automatically release the internal ByteBuf
+     * @return
+     */
     public byte[] getEntry() {
+        if (data == null) {
```

Comment:
I don't understand here:

why do you need this change here? it doesn't seem to be necessary.